include/wil/stl.h

```diff
@@ -172,6 +171,15 @@ namespace wil
         }
 
     private:
+        // Bounds-checked version of char_traits::length, like strnlen. Requires that the input contains a null terminator.
+        static constexpr size_type length_n(_In_reads_opt_(length) const TChar* str, size_type buf_size) noexcept

```

Comment:
One last minor comment. The `_In_reads_opt_(  --->  length  <---  )` does not match the `buf_size` argument nmae